joomla/joomla-cms
# [4.0] [com_finder] Disable search statistics by default

### Summary of Changes

In the administrator part of Smart Search, search statistics option appears to be disabled but in frontend statistics are still gathered. This changes default value of statistics option in frontend to disabled.

Also renames `logging_enabled` parameter to `gather_search_statistics` to avoid confusion with `enable_logging`.

### Testing Instructions

In backend go to Components -> Smart Search -> Statistics.
Notice the warning:

>Gathering of statistics is disabled. Enable it in the Options.

Go to frontend. Perform a search using Smart Search.
Go back to statistics page.

### Expected result

No entries logged.

### Actual result

Search entry logged.

### Documentation Changes Required

No.
